assets/scss/design-system.scss

```
@@ -0,0 +1,1435 @@
+	--sc-input-value-font-weight: var(--sc-font-weight-semibold);
+	--sc-input-value-color: var(--sc-color-dark-gray);
+	--sc-input-value-letter-spacing: 0;
+
+	/* ===== SIZING TOKENS ===== */
+	--sc-radius-sm: 10px;
+	--sc-radius-md: 15px;
+	--sc-radius-lg: 16px;
+	--sc-icon-size: 24px;
+	--sc-icon-xs-size: 13px;
+
+	/** ------------FONT SIZE VARIABLES-------------*/
```

Stylelint (17.7.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)